src/api/monthly-curation.ts

```diff
@@ -1,8 +1,30 @@
 import { httpClient } from './index';
 
+export type ItemList = {
+  brand: string;
+  description: string;
+  id: number;
+  imgUrl: string;
+  like: boolean;
+  name: string;
+  procie: number;
+};
+
+export type MonthlyCurationResponse = {
+  imgUrl: string | null;
+  itemList: ItemList[];
+  month: number;
+  subtitle: string;
+  title: string;
+  year: number;
+};
+
 export const monthlyCuration = async (year: number, month: number) => {
-  const { data } = await httpClient.get(`/public/monthly/curation/${year}/${month}`, {
-    params: { year, month },
-  });
+  const { data } = await httpClient.get<MonthlyCurationResponse[] | any>(
```

Comment:
any 타입은 제거해도 좋을 것 같습니다.
```suggestion
  const { data } = await httpClient.get<MonthlyCurationResponse[]>(
```